render_pdf/src/parser.py

```diff
@@ -0,0 +1,129 @@
+from lxml import etree
```

Comment:
One of our transitive dependencies uses lxml, but let's use `xml.etree.ElementTree` instead, which has 99% of the functionality and is also in the standard lib. grep in `ambuda` for examples

Reply:
I'd rather not rewrite; any specific reason why you want to move to lxml? If possible, I'll leave this rewrite for another PR.